webapp/src/components/legal_holds_setting.tsx

```diff
@@ -99,8 +110,8 @@ const LegalHoldsSetting = () => {
             <div
                 style={{
                     padding: '28px 32px',
-                    border: '1px solid rgba(var(--sys-center-channel-color-rgb), 0.08)',
-                    background: 'var(--sys-center-channel-bg)',
+                    border: '1px solid rgba(0, 0, 0, 0.08)',
+                    background: '#ffffff',
```

Comment:
Why aren't we using the theme variables here?

Reply:
Probably commited by accident after testing something. Reverting.